src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/199233739-53bc8cc1-5d57-4f75-923b-25794cb16f0d.png)

Reply:
I had fixed all the mistakes that I saw according to checklist, homework is ready to watch